cv_next/app/cv/[cvId]/components/commentSection/comment.tsx

```diff
@@ -387,9 +387,9 @@ export default function Comment({
           <p className="mr-3 inline-flex items-center text-sm font-semibold text-gray-900 dark:text-white">
             <Link
               className="text-lg font-medium hover:underline"
-              href={`/profile/${commenter.id}`}
+              href={`/profile/${commenter.username}`}
             >
-              {commenter.username || commenter.full_name}
+              {commenter.full_name}
```

Comment:
Can we also change the field name to `display_name` instead of `full_name`? It'll be more consistent with what we're trying to convey with this field.